Core.BackgroundJobs/Model/ReadModels/ScheduleItSystemUsageOverviewReadModelUpdates.cs

```diff
@@ -39,7 +39,12 @@ public class ScheduleItSystemUsageOverviewReadModelUpdates : IAsyncBackgroundJob
         public Task<Result<string, OperationError>> ExecuteAsync(CancellationToken token = default)
         {
             var updatesExecuted = 0;
-            var alreadyScheduledIds = new HashSet<int>();
+            var idsOfItSystemUsagesAlreadyInQueueForReadModelUpdate = _updateRepository
```

Comment:
vi cacher lige alle dem der allerede er i kø som udgangspunkt for vores filtrering